docs/operating-eck/eck-permissions.asciidoc

```diff
@@ -8,8 +8,12 @@ endif::[]
 [id="{p}-{page_id}"]
 = Required RBAC permissions
 
-Installing and running ECK, as well as using ECK-managed resources requires certain Kubernetes link:https://kubernetes.io/docs/reference/access-authn-authz/rbac[permissions]. These permissions are listed below, with a reason why they are needed.
+Installing and running ECK, as well as using ECK-managed resources requires the following Kubernetes link:https://kubernetes.io/docs/reference/access-authn-authz/rbac[permissions]:
 
+* <<{p}-installing-crds,Installing CRDs>> 
+* <<{p}-installing-operator,Installing the ECK operator>> 
+* <<{p}-running,Running ECK operator>> 
+* <<{p}-using,Using ECK-managed resources>> 
```

Comment:
I'm not sure why we inserted `{page_id}` in the ids of the sections of this page.
We can either update this list with the `{page_id}`:
```suggestion
* <<{p}-{page_id}-installing-crds,Installing CRDs>> 
* <<{p}-{page_id}-installing-operator,Installing the ECK operator>> 
* <<{p}-{page_id}-running,Running ECK operator>> 
* <<{p}-{page_id}-using,Using ECK-managed resources>> 
```

Or remove `{page_id}` from the ids of the section and then updates the references (there are some in `docs/operating-eck/eck-permissions.asciidoc` and `docs/operating-eck/installing-eck.asciidoc`).
```diff
--- a/docs/operating-eck/eck-permissions.asciidoc
+++ b/docs/operating-eck/eck-permissions.asciidoc
@@ -10,13 +10,13 @@ endif::[]
 
 Installing and running ECK, as well as using ECK-managed resources requires the following Kubernetes link:https://kubernetes.io/docs/reference/access-authn-authz/rbac[permissions]:
 
-* <<{p}-installing-crds,Installing CRDs>> 
-* <<{p}-installing-operator,Installing the ECK operator>> 
-* <<{p}-running,Running ECK operator>> 
-* <<{p}-using,Using ECK-managed resources>> 
+* <<{p}-installing-crds,Installing CRDs>>
+* <<{p}-installing-operator,Installing the ECK operator>>
+* <<{p}-running,Running ECK operator>>
+* <<{p}-using,Using ECK-managed resources>>
 
 [float]
-[id="{p}-{page_id}-installing-crds"]
+[id="{p}-installing-crds"]
 == Installing CRDs
 
 This permission is required to install CRDs. CRDs (link:https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/[CustomResourceDefinitions]) are the only non-namespaced resources required to be installed.
@@ -28,7 +28,7 @@ This permission is required to install CRDs. CRDs (link:https://kubernetes.io/do
 |===
 
 [float]
-[id="{p}-{page_id}-installing-operator"]
+[id="{p}-installing-operator"]
 == Installing the ECK operator
 
 These permissions are required to install the ECK operator in a Kubernetes cluster.
@@ -47,10 +47,10 @@ These permissions are required to install the ECK operator in a Kubernetes clust
 |Service|core|yes|Service for validating webhook endpoint.
 |===
 
-And all permissions that <<{p}-{page_id}-running>> section specifies.
+And all permissions that <<{p}-running>> section specifies.
 
 [float]
-[id="{p}-{page_id}-running"]
+[id="{p}-running"]
 == Running ECK operator
 
 These permissions are needed by the Service Account that ECK operator runs as.
@@ -73,10 +73,10 @@ These permissions are needed by the Service Account that ECK operator runs as.
 |coreauthorization.k8s.io|SubjectAccessReview|yes|Controlling access between referenced resources. Check link:https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-restrict-cross-namespace-associations.html[docs] to learn more.
 |===
 
-And all permissions that the <<{p}-{page_id}-using>> chapter specifies.
+And all permissions that the <<{p}-using>> chapter specifies.
 
 [float]
-[id="{p}-{page_id}-using"]
+[id="{p}-using"]
 == Using ECK-managed resources
 
 These permissions are needed to manage each Elastic Stack application. For example, to create, update and delete Elasticsearch clusters the permissions for the respective verbs must be held by the user that performs the operation.
diff --git a/docs/operating-eck/installing-eck.asciidoc b/docs/operating-eck/installing-eck.asciidoc
index 9eaa058ce..eb24f4890 100644
--- a/docs/operating-eck/installing-eck.asciidoc
+++ b/docs/operating-eck/installing-eck.asciidoc
@@ -7,7 +7,7 @@ endif::[]
 [id="{p}-{page_id}"]
 = Install ECK 
 
-{n} (ECK) is a link:https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Kubernetes operator] to orchestrate Elastic applications ({eck_resources_list}) on Kubernetes. It relies on a set of link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions[Custom Resource Definitions (CRD)] to declaratively define the way each application is deployed. CRDs are global resources shared by all users of the Kubernetes cluster, which requires <<{p}-eck-permissions-installing-crds,certain permissions>> to install them for initial use. The operator itself can be installed as a cluster-scoped application managing all namespaces or it can be restricted to a pre-defined set of namespaces. Multiple copies of the operator can be installed on a single Kubernetes cluster provided that the global CRDs are compatible with each instance and optional cluster-scoped extensions such as the <<{p}-webhook,validating webhook>> are disabled.
+{n} (ECK) is a link:https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Kubernetes operator] to orchestrate Elastic applications ({eck_resources_list}) on Kubernetes. It relies on a set of link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions[Custom Resource Definitions (CRD)] to declaratively define the way each application is deployed. CRDs are global resources shared by all users of the Kubernetes cluster, which requires <<{p}-installing-crds,certain permissions>> to install them for initial use. The operator itself can be installed as a cluster-scoped application managing all namespaces or it can be restricted to a pre-defined set of namespaces. Multiple copies of the operator can be installed on a single Kubernetes cluster provided that the global CRDs are compatible with each instance and optional cluster-scoped extensions such as the <<{p}-webhook,validating webhook>> are disabled.
```

Reply:
Thanks @thbkrkr, I'll fix the references by adding `{page_id}`.